render-test/parser.cpp

```diff
@@ -273,6 +273,21 @@ std::string serializeMetrics(const TestMetrics& metrics) {
     rapidjson::Writer<rapidjson::StringBuffer> writer(s);
 
     writer.StartObject();
+
+    // Start fileSize section.
+    writer.Key("fileSize");
```

Comment:
Do we need to add empty keys if there are no metrics? There is a check when json is parsed `if (document.HasMember("fileSize")) {`.

Reply:
We do it for `memory`. Probably we should not do it because rebaselines will get noisy when we add a new probe.